docs/configuring/chargebee-integration.md

```diff
@@ -0,0 +1,47 @@
+#Chargebee integration
+
+KuberLogic uses Webhooks to interact with the billing providers (such as Chargebee).
```

Comment:
I think we should describe what chargebee integration can do (provision instances and that's it)